CRM/Extension/Browser.php

```diff
@@ -221,12 +177,8 @@ private function _discoverRemote() {
    * @return string
    */
   private function grabCachedJson() {
-    $filename = $this->cacheDir . DIRECTORY_SEPARATOR . self::CACHE_JSON_FILE . '.' . md5($this->getRepositoryUrl());
-    $json = NULL;
-    if (file_exists($filename)) {
-      $json = file_get_contents($filename);
-    }
-    if (empty($json)) {
+    $json = \Civi::cache('extension_browser')->get('extdir');
```

Comment:
Fun fact: There can be multiple URLs, eg

* The sysadmin can configure the `ext_repo_url` setting (eg to enable alpha/beta downloads).
* The sysadmin can use CLI options in `cv dl` (like `--dev`, `--filter-ver`, `--filter-status`) which grab from another feed.

As in the `$filename` above, the cache-key (`get($key)` and `set($key,$value)`) should include the URL. Something like:

```php
$json = \Civi::cache('extension_browser')->get('extdir_' . $this->getRepositoryUrl());
$json = \Civi::cache('extension_browser')->get('extdir_' . md5($this->getRepositoryUrl()));
$json = \Civi::cache('extension_browser')->get(md5($this->getRepositoryUrl()));
```